code/_onclick/other_mobs.dm

```diff
@@ -29,12 +33,16 @@
 	return
 
 /mob/living/carbon/human/RangedAttack(var/atom/A)
-	if(!gloves && !mutations.len) return
-	var/obj/item/clothing/gloves/G = gloves
+	if(!(gloves || mutations.len || glasses)) return
```

Comment:
I'm curious now. Has this code been tested? It seems awfully dubious: you require all three entities to pass this check.

Reply:
It requires having at least one. De Morgan's Theorem, !A & !B & !C == !(A | B | C).

Reply:
Oh, and it's been tested too.